modules/lang-mustache/src/main/java/org/elasticsearch/script/mustache/SearchTemplateRequestBuilder.java

```diff
@@ -47,6 +47,16 @@ public SearchTemplateRequestBuilder setSimulate(boolean simulate) {
         return this;
     }
 
+    public SearchTemplateRequestBuilder setExplain(boolean explain) {
```

Comment:
Sure. I added to the RequestBuilder. I guess we don't need to document the Request itself, should we?